examples/distinct-object-types/distinct_object_types.bal

```diff
@@ -0,0 +1,36 @@
+import ballerina/io;
+
+// The `Person` object type that contains a string field called `name`.
```

Comment:
Shall we remove `The` from the type comments

Reply:
Updated d8c6a9e496c62f91e5499baaa4413757c01d639c

Reply:
Better to add it for grammatical correctness if we don't have a strong reason to remove it.